src/everest/templates/__init__.py

```diff
@@ -1,11 +1,17 @@
-import pkg_resources
+import importlib
+from pathlib import Path
 
 template_names = (
+    "schmerge.tmpl",
```

Comment:
Why is this needed?

Reply:
I dont know why the template_names list is needed. Either it should include all templates in the dir or we should remove it